src/pilot/deploy-overcloud.py

```diff
@@ -398,7 +1207,6 @@ def main():
               " --os-user-id {}" \
               " --os-password {}" \
               " --control-scale {}" \
-              " --compute-scale {}" \
```

Comment:
Is there no other way to do this and be more consistant and rather that edit the environment file pass in a parameter to reference the custom role here ? ( for manual deployments )